README.md

```diff
@@ -40,7 +40,7 @@ Writing security-conscious IAM Policies by hand can be very tedious and ineffici
 
  * Determined to make your best effort to give users and roles the least amount of privilege you need to perform your duties, you spend way too much time combing through the AWS IAM Documentation on [Actions, Resources, and Condition Keys for AWS Services][1].
  * Your team lead encourages you to build security into your IAM Policies for product quality, but eventually you get frustrated due to project deadlines.
- * You don't have an embedded security person on your team who can write those IAM policies for you, and there's no automated tool that will automagically sense the AWS API calls that you perform and then write them for you in a least-privilege manner.
+ * You don't have an embedded security person on your team who can write those IAM policies for you, and there's no automated tool that will [automagically sense the AWS API calls that you perform](https://github.com/princespaghetti/actionhero) and then write them for you in a least-privilege manner.
```

Comment:
Nit: I don't want to include that as a link here as it detracts from the point.

I did look at ActionHero back then, but my issue with it was that (1) it missed a lot of API Calls (this is AWS's fault because there is not a 1-1 mapping between CloudTrail event names, API call names, IAM namespaces) and (2) it doesn't include resource ARNs.

Since then, Ian McCay's [iamlive](https://github.com/iann0036/iamlive) tool has come out which has a ridiculously accurate mapping of the IAM/CloudTrail/API names, and even identifies undocumented APIs. I **love** iamlive. However, it still doesn't support resource ARNs due to limitations of AWS Client side monitoring.

Therefore, I think it would be fair to suggest that we change this to "there's no automated tool that will automagically sense the AWS API calls that you perform and then write them for you with resource ARN constraints." I can approve that if you make that change :)

Reply:
I actually just found `iamlive` last night after looking for more CSM tools so didn't know of it when I wrote the PR! Agreed with changing the wording given those details now and not having a link

Reply:
modified wording as discussed